src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper
+        implements DtoResponseMapper<UserResponseDto, User> {
+    @Override
+    public UserResponseDto toDto(User user) {
+        UserResponseDto responseDto = new UserResponseDto();
+        responseDto.setId(user.getId());
+        responseDto.setEmail(user.getEmail());
+        responseDto.setPassword(user.getPassword());
```

Comment:
It's not a good practice to include sensitive information such as password and salt in the UserResponseDto. Consider removing these fields from the response object.